8_child_illness.do

```diff
@@ -76,7 +76,13 @@ c_illtreat
 			replace c_treatdiarrhea = 1 if c_treatdiarrhea == 0 & ca7a == 1	// received ORS
 			replace c_treatdiarrhea = . if cage == . | ca1 != 1   
 			replace c_treatdiarrhea = . if inlist(ca7a, 8, 9)      // missing when both ORS variables are missing and none o	
-		}		
+		}
```

Comment:
could consider adding this section back to the list of countries ending with line 67.

Reply:
checked - indeed two sections are identical

Reply:
@Ortsang 
Sorry man, I messed up on the logic here. Note in line 71, both ca7a and ca7b are required for the program to execute. However, for Tonga2019, we don't have ca7a. 

It appears your previous approach (having a separate chunk, preserving only ca7b) was right. Could you please revert relevant lines in file 8_child illness.do back to what it was before?

I just noticed the merged version prevented the program from running through in its entirety. Sorry for the inconvenience

Reply:
checked;
1. tonga2019 listed removed;
2. tonga2019 specific codes added